tests/test_register.py

```diff
@@ -46,7 +47,8 @@ def test_successful_register(register_settings):
 
 
 def test_exception_for_redirect(register_settings):
-    """Test a repository url redirection during registration"""
+    """Package registration fails if the repository URL
```

Comment:
Same thing here. Further, the first line should fit on one-line, including a period and additional information should be specified after a blank line in paragraph form.